config/session.php

```diff
@@ -16,7 +16,7 @@
     |
     */
 
-    'driver' => env('SESSION_DRIVER', 'array'),
+    'driver' => env('SESSION_DRIVER', 'file'),
```

Comment:
not that we have to do it now, but we should definitely see if we can get [redis](http://redis.io/) to be our session driver